src/Cassandra/UdtMap.cs

```diff
@@ -83,7 +84,9 @@ public abstract class UdtMap
         protected readonly Dictionary<PropertyInfo, string> _propertyToFieldName;
         // ReSharper enable InconsistentNaming
         private Serializer _serializer;
-        protected const BindingFlags PropertyFlags = BindingFlags.FlattenHierarchy | BindingFlags.IgnoreCase | BindingFlags.Public | BindingFlags.Instance;
+        public const BindingFlags PropertyFlags = BindingFlags.FlattenHierarchy | BindingFlags.IgnoreCase | BindingFlags.Public | BindingFlags.Instance;
+
+        public TypeConverter TypeConverter { get; private set; }
```

Comment:
I'm thinking about creating a new class UdtFieldConverter that uses a TypeConverter passed in the constructor and that would expose two methods : ConvertToDbUdtFieldValue() and ConvertToUdtFieldDbValue(). 
An instance of that class that uses a DefaultTypeConverter would be exposed in the MappingConfiguration (just like the MapperFactory).
When ConvertTypesUsing(TypeConverter typeConverter) is called in the MappingConfiguration, it would change the TypeConverter used by the UdtFieldConverter as well.

Is that design better?